.readthedocs.yml

```diff
@@ -1,15 +1,42 @@
+# version: 2
```

Comment:
If this is working correctly you can remove all the commented lines in this file